src/attached_probe.cpp

```diff
@@ -960,11 +962,39 @@ void AttachedProbe::attach_kprobe(bool safe_mode)
     return;
   }
 
+  // Construct a string containing "module:function."
+  // Also throw a warning or error if the module doesn't exist,
+  // before attempting to attach.
+  // Note that we do not pass vmlinux, if it is specified.
+  std::string funcname = probe_.attach_point;
+  std::string modname = probe_.path;
+  if ((modname.length() > 0) && (modname.compare("vmlinux") != 0))
+  {
+    if (!is_loaded_module(modname))
+    {
+      if (probe_.orig_name != probe_.name)
+      {
+        // Wildcard usage just gets a warning
+        LOG(WARNING) << "module " << modname << " in probe " << probe_.name
```

Comment:
If the module isn't loaded - how do we know it doesn't exist? Rather why doesn't this mirror the error message below?

Reply:
It should definitely mirror the below error message.